examples/ConfigOnDoubleReset/ConfigOnDoubleReset.ino

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
I have devices like this , to reset or force config we have to press reset within the first 5 seconds ( a status light blinks ) and do so 5 times in a row, it is a pretty decent method to prevent accidental glitch wipes etc. ( this is commercial equipment )